packages/app-builder-lib/src/appInfo.ts

```diff
@@ -39,6 +40,7 @@ export class AppInfo {
     normalizeNfd = false
   ) {
     this.version = info.metadata.version!
+    this.type = info.metadata.type!
```

Comment:
What happens if `type` is not specified in the `package.json`? Seems we should leave this as optional?

Reply:
@mmaietta updated. Any insights why `pull_request_target` is failing?